lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view_web.dart

```diff
@@ -547,7 +547,8 @@ class MailboxDashBoardView extends BaseMailboxDashBoardView {
                   controller.searchEmail(context, recent.value);
                 },
                 suggestionsCallback: (pattern) async {
-                  return controller.quickSearchEmailsAction(pattern);
+                  controller.patternSearch.value = pattern;
+                  return controller.searchEmailsAction();
```

Comment:
I am not a fan of it. We should encourage functional programming (modify the `state` out of the function as little as possible